pkg/types/alibabacloud/platform.go

```diff
@@ -7,11 +7,32 @@ type Platform struct {
 
 	// ResourceGroupID is the ID of an already existing resource group where the cluster should be installed.
 	// This resource group must be empty with no other resources when trying to use it for creating a cluster.
-	// If empty, a new resource group will created for the cluster.
-	// Destroying the cluster using installer will delete this resource group.
+	// If empty, a new resource group will created for the cluster,
+	// and destroying the cluster using installer will delete this resource group.
```

Comment:
This change states that the resource group will only be when the user let the installer create the resource group. Instead, the resource group is *always* deleted, regardless of whether it was user-supplied or installer-created.

Reply:
Moved this to #5378.